api/c/indigo/src/indigo_options.cpp

```diff
@@ -343,12 +342,11 @@ void IndigoOptionHandlerSetter::setBasicOptionHandlers(const qword id)
     mgr->setOptionHandlerBool("standardize-clear-unknown-stereo", SETTER_GETTER_BOOL_OPTION(indigo.standardize_options.clear_unknown_stereo));
     mgr->setOptionHandlerBool("standardize-clear-unknown-atom-stereo", SETTER_GETTER_BOOL_OPTION(indigo.standardize_options.clear_unknown_atom_stereo));
     mgr->setOptionHandlerBool("standardize-clear-unknown-bond-stereo",
-                             SETTER_GETTER_BOOL_OPTION(indigo.standardize_options.clear_unknown_cis_trans_bond_stereo));
+                              SETTER_GETTER_BOOL_OPTION(indigo.standardize_options.clear_unknown_cis_trans_bond_stereo));
```

Comment:
It makes sense to make an exception for this table-like block of code and not to break lines.

Reply:
It's done automatically by clang-format based on max line length. Too long lines are bad for readability.